### ISSUE TITLE
Scrum Poker functionality

### ISSUE THREAD

**Issue by mmospanenko** [matterpoll/matterpoll#523]
#### Hide while vote is in progress

Will be great to have ability to hide participants while result is not open, very useful for Poker Planing.

Like StoryPlan has

**Comment by hanzei** [comment]
I don't quite understand the request. By default, a poll does show who voted for what _after_  the poll has ended.

**Comment by mmospanenko** [comment]
yes, we're using it as is, it works, will be great to have some predefined configuration like Fibonacci answers list, etc
